client/src/components/BrowserWarning/BrowserWarning.js

```diff
@@ -0,0 +1,6 @@
+import browser from 'detect-browser';
+import $ from 'jquery';
+
+if (browser.name === 'ie' && parseInt(browser.version, 10) <= 10) {
+  $('.browser-warning-wrapper').addClass('browser-warning-wrapper--show');
+}
```

Comment:
I think this would be more suited in the legacy folder than inter-mingled with the components, or even the lib folder.
This doesn't have any "component" exported.